client/lib/signup/site-type.js

```diff
@@ -102,6 +102,7 @@ export function getAllSiteTypes() {
 			domainsStepSubheader: i18n.translate(
 				"Enter your blog's name or some keywords that describe it to get started."
 			),
+			hideSiteMockups: true,
```

Comment:
Just noting there's a dependency between this and https://github.com/Automattic/wp-calypso/pull/34479 - before merging one of these PRs we need to keep the other one in mind